lib/galaxy_test/api/test_workflows.py

```diff
@@ -7102,9 +7102,9 @@ def test_invocations_accessible_imported_workflow(self):
             self._assert_status_code_is(other_import_response, 200)
             other_id = other_import_response.json()["id"]
             workflow_request, history_id, _ = self._setup_workflow_run(workflow_id=other_id)
-            response = self._get(f"workflows/{other_id}/usage")
-            self._assert_status_code_is(response, 200)
-            assert len(response.json()) == 0
+            # response = self._get(f"workflows/{other_id}/usage")
+            # self._assert_status_code_is(response, 200)
+            # assert len(response.json()) == 0
```

Comment:
```suggestion
            response = self._get(f"workflows/{other_id}/usage")
            self._assert_status_code_is(response, 200)
            assert len(response.json()) == 0
```
Not sure why you've commented these tests ?

Reply:
They are deprecated but this was never formally announced. We can mark these as deprecated with `deprecated=True` in the router function and then consider removing them, but for now I think we still need them.